app/interface/deps/services/__init__.py

```
@@ -0,0 +1,4 @@
+from .auth_deps import get_jwt_service, get_signup_service
+from .course_deps import get_all_courses_detail_service
+
+__all__ = ["get_jwt_service", "get_signup_service", "get_all_courses_detail_service"]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)